src/Stakeholders/components/Roles/ListItem/index.js

```diff
@@ -135,21 +135,27 @@ class RoleListItem extends Component {
       >
         <Row>
           <Col span={1}>{sideComponent}</Col>
-          <Col span={7}>{name}</Col>
-          <Col span={3}>{abbreviation}</Col>
-          <Col span={10}>{description}</Col>
+          <Col span={7} title="Role of focal people">
+            {name}
+          </Col>
+          <Col span={3} title="role abbreviation ">
+            {abbreviation}
+          </Col>
+          <Col span={10} title="An additional information attached to a role">
+            {description}
+          </Col>
           <Col span={3}>
             {isHovered && (
               <Fragment>
                 <Icon
                   type="edit"
-                  title="Update Role"
+                  title="Click to update Role"
```

Comment:
was this supposed to be changed?

Reply:
yes lally told me to put that way

Reply:
ok I talked to Lally so can we return these back to original titles without click word?